content/en/docs/Tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,83 @@
+# Admission Webhooks
+
+## Background
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator.
+
+You may be unfamiliar with webhooks. At a high level, after a request has been authenticated and authorized, admission webhooks intercept requests against the Kubernetes API and have an opportunity to validate or update the object before it is saved in the object store. Please refer to the following table that highlights what each webhook is capable of:
+
+|                    | Validating Webhook | Mutating Webhook |
+|--------------------|--------------------|------------------|
+| Validating Objects |          x         |         x        |
+| Mutating Objects   |                    |         x        |
+
+If you are interested in learning more about admission webhooks, please review the [official kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-are-they).
+
+## Defining your Webhook in the ClusterServiceVersion
```

Comment:
This seems like a good place to start the doc. (Context: comment about moving the `Background` section to `References`